src/components/Editor.js

```diff
@@ -490,7 +490,7 @@ const Editor = React.createClass({
     this.editor.destroy();
     this.editor = null;
 
-    const searchAgainKey = L10N.getStr("sourceSearch.search.again.key");
+    const searchAgainKey = L10N.getStr("sourceSearch.search.next.key");
```

Comment:
There are two instances of this.  We need to change the other one as well.

https://github.com/jasonLaster/debugger.html/blob/387bc86e4302c99feeda3988915df5dccf313ceb/src/components/Editor.js#L475